test/IntegrationTests/IntegrationTests.csproj

```diff
@@ -4,12 +4,21 @@
     <AddTestStrongNameAssemblyKeyOnNetFramework>true</AddTestStrongNameAssemblyKeyOnNetFramework>
     <!-- No warn is needed for autogenerated code by protobuf: CS8981 - The type name only contains lower-cased ascii characters. -->
     <NoWarn Condition="'$(TargetFramework)' == 'net8.0' OR '$(TargetFramework)' == 'net7.0'">CS8981</NoWarn>
+    <Platforms>x64;ARM64</Platforms>
```

Comment:
Is this really needed?

Reply:
Yes, this is connected to _X64 and _ARM64 constant. Visual Studio is using a `$(Platform)` variable to define these constants. It also makes Visual Studio interacting with the code that is marked as `#if _X64` so as soon as you select x64 the code has intellisense.

Reply:
Instead of assigning a Platform here and in the Nuke code, can we 1) compile these as AnyCPU, 2) remove the ifdefs, and 3) check the architecture dynamically when the test is run and skip the test then?

The reason I ask is that the solution file is building this project as AnyCPU for release builds but platform-specific for debug builds, and I'd like to build managed code as AnyCPU as much as possible.

Reply:
The only issue with runtime check was that it has to be done in multiple places. 
1. In the test itself, need to change Theory to SkippableTheory and then skip the test if arm64
2. In the `SqlServerFixture` (or any fixture that does not work) cause this executes way before the test and fails there.

Is a bit more fragmented approach but still not that complicated to do. Personally don't have strong feelings about either of the approach.

Reply:
I like @zacharycmontoya suggestion, but, it can come in a follow up.

Reply:
I think @Kielek was currently the only one who preferred more compile time filter than runtime... which is definitely cleaner solution. But up to you... I'd prefer to do it in the current branch (while there is history to check just in case)

Reply:
From my perspective, it is just a preference. Any working options should be fine :)

Reply:
Sounds good, it's good to merge and we can perform this as follow-up 👍🏼